pyproject.toml

```diff
@@ -10,7 +10,7 @@ python = "^3.8,<3.11"
 blessings = "^1.7"
 coloredlogs = "^15.0.1"
 demisto_py = "^3.1.6"
-demisto_sdk= "^1.6.8"
+demisto_sdk = {git = "https://github.com/demisto/demisto-sdk.git", rev = "1.6.8_coveragefix"}
```

Comment:
in general, we are not using a branch for hotfixes (just the commit id).
this is used to avoid SDK changes without a PR in content.

Reply:
changed